packages/theming/theme-types/src/Color.types.ts

```diff
@@ -451,152 +451,362 @@ export interface ControlColorTokens {
 }
 
 export interface AliasColorTokens {
+  /// Neutral colors
+
+  /** Defined on platforms: android, iOS, macOS, win32, windows */
   neutralForeground1: ColorValue;
+
+  /** Defined on platforms: macOS, win32, windows */
```

Comment:
Realized maybe I should delete all the empty lines in between each token definition, to be consistent with some comments above

Reply:
I personally like the empty lines better fwiw

Reply:
Same. It's more readable.